install/ossm-istio-ambient-mode.adoc

```diff
@@ -0,0 +1,37 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ossm-istio-ambient-mode"]
+= Istio ambient mode
+include::_attributes/common-attributes.adoc[]
+:context: ossm-istio-ambient-mode
+
+toc::[]
+
+include::snippets/technology-preview-istio-ambient-mode.adoc[]
+
+{istio} ambient mode introduces a sidecar-less architecture for {SMProductName}, designed to simplify operations and reduce resource usage. Instead of injecting a sidecar proxy into each application pod, ambient mode uses a shared node-level proxy for Layer 4 (L4) functionality and an optional, dedicated proxy for Layer 7 (L7) features.
+
+include::modules/ossm-about-istio-ambient-mode.adoc[leveloffset=+1]
+include::modules/ossm-installing-istio-ambient-mode.adoc[leveloffset=+2]
+
+.Next steps
+* xref:../install/ossm-istio-ambient-mode.adoc#ossm-scoping-sm-discovery-selectors-istio-ambient-mode_ossm-istio-ambient-mode[Scoping Service Mesh with discovery selectors in Istio ambient mode]
+* xref:../install/ossm-istio-ambient-mode.adoc#ossm-deploying-bookinfo-application-istio-ambient-mode_ossm-istio-ambient-mode[Deploying the Bookinfo application in Istio ambient mode]
+
+include::modules/ossm-about-discovery-selectors-istio-ambient-mode.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../install/ossm-installing-openshift-service-mesh.adoc#ossm-scoping-service-mesh-with-discoveryselectors_ossm-creating-istiocni-resource[Scoping the Service Mesh with discovery selectors]
+
+include::modules/ossm-scoping-sm-discovery-selectors-istio-ambient-mode.adoc[leveloffset=+2]
+include::modules/ossm-deploying-bookinfo-application-istio-ambient-mode.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../install/ossm-installing-openshift-service-mesh.adoc#ossm-about-bookinfo-application_ossm-discoveryselectors-scope-service-mesh[About the Bookinfo application]
+
+[role="_additional-resources"]
+[id="additional-resources_{context}"]
+== Additional resources
+* link:https://istio.io/latest/docs/ambient/architecture/[Ambient mode architecture] (Istio documentation)
+* link:https://istio.io/latest/docs/ambient/usage/add-workloads/[Adding workloads to a mesh in ambient mode] (Istio documentation)
```

Comment:
All additional resources list item text must be in the link text string (this is a new tweak to support docs migration, you might not have heard yet)

```suggestion
* link:https://istio.io/latest/docs/ambient/architecture/[Ambient mode architecture (Istio documentation)]
* link:https://istio.io/latest/docs/ambient/usage/add-workloads/[Adding workloads to a mesh in ambient mode (Istio documentation)]
```